docs/DeveloperGuide.md

```diff
@@ -3,39 +3,97 @@ layout: page
 title: Developer Guide
 ---
 
-# UniCa$h Developer Guide [CS2103 T16-Group_3 TP]
+**Team:** T16-03
 
 **Name:** UniCa$h
 
-**User Target Profile:** Our application is for university students who want to be more financially conscious about their spending habits, to enable them to make more economical decisions that provide students on a limited budget with more purchasing power.
+**User Target Profile:** Our application is for university students who want to be more financially conscious about their spending habits, enabling them to make wiser economic decisions, providing students on a limited budget with more purchasing power.
 
-**Value Proposition:** It provides university students with an intuitive and frictionless experience to have transparency on their expenditure to help them better understand their spendings so they can develop better financial habits as they transition into adulthood. 
+**Value Proposition:** It provides university students with an intuitive and frictionless experience to have transparency on their expenditure to help them better understand their expenditure so they can develop better financial habits as they transition into adulthood.
+
+{% include toc.html %}
+
+## Terminology
+
+| Term        | Meaning                                                                                           |
+|-------------|---------------------------------------------------------------------------------------------------|
+| Transaction | Represents both an expense or an income. Expenses cause a net loss while incomes cause a net gain |
 
 ## Potential Features:
-- For every user > track individual finances (base)
-- Input spending & set budgets (base)
-- Tag another user and distribute the spendings during outings (extension)
-- Viewing spending habits (extension)
-- Provide price list comparison for all available food outlets near NUS (extension)
-- Allow users to set repeating events (extension)
-- Tagging people for payments gives them a notification (extension)
-
-## Feature List [v1.2]
+
+Potential features are split into two categories, must-haves and good-to-haves. Our focus this project was to flesh out the must-haves first.
+
+### Must-haves
+
+Every user can...
+
+- Track and manage their expenses
+- Manage a budget
+- View their total expenditure by the month/year
+
+### Good-to-haves
+
+Every user can...
+
+- Visualise their spending habits
+- Divide a group spending and tag friends involved in the spending
+- View a price list comparison of all available shops near NUS
+- Set recurring transactions
```

Comment:
Just a suggestion to improve this segment. Do you think it'll be better if we add tick marks next to these "Must-haves" and "Good-to-haves" to indicate which of these have been implemented? Nothing wrong with the current bullet-points though! 😄

Reply:
I believe we have implemented all of the must-haves but only one of the good-to-haves. I'm not sure if having ticks will make them think we're missing features...